Tests/AK/TestBitmap.cpp

```diff
@@ -152,28 +152,28 @@ TEST_CASE(set_range)
 TEST_CASE(find_first_fit)
 {
     {
-        Bitmap bitmap(32, true);
+        Bitmap bitmap = Bitmap::must_create_but_fixme_should_propagate_errors(32, true);
         auto fit = bitmap.find_first_fit(1);
         EXPECT_EQ(fit.has_value(), false);
     }
     {
-        Bitmap bitmap(32, true);
+        Bitmap bitmap = Bitmap::must_create_but_fixme_should_propagate_errors(32, true);
```

Comment:
We should probably make the tests use the try variant and wrap them in MUST, as we don't really care / need to / have a way to propagate errors out of tests. Maybe we could make a `EXPECT_TRY` test macro for this scenario?